src/test/resources/com/puppycrawl/tools/checkstyle/api/violation/InputViolation1.java

```diff
@@ -0,0 +1,22 @@
+/*
+com.puppycrawl.tools.checkstyle.checks.modifier.InterfaceMemberImpliedModifierCheck
+violateImpliedPublicField = false
+violateImpliedStaticField = (default)true
+violateImpliedFinalField = (default)true
+violateImpliedPublicMethod = (default)true
+violateImpliedAbstractMethod = (default)true
+violateImpliedPublicNested = (default)true
+violateImpliedStaticNested = (default)true
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.api.violation;
+
+public interface InputViolation1 {
```

Comment:
name it InputTwoViolationOnSameLine

Reply:
https://github.com/checkstyle/checkstyle/pull/13810#discussion_r1342145428